src/lib/init.ts

```diff
@@ -790,6 +813,77 @@ function getGatewayPackageJSON(isJS: boolean) {
   };
 }
 
+function getGatewayServerless(isJS: boolean) {
+  return `${
+    isJS
+      ? ""
+      : 'import type { TypeBoxTypeProvider } from "@fastify/type-provider-typebox";\n'
+  }import fastify from "fastify";
+import dotenv from "dotenv";
+
+// Runtime dependencies required for this function. Vercel does import only direct listed dependencies.
+(() => [
+  import("@fluencelabs/js-client"),
+  // This import will fail in runtime
```

Comment:
I don't know what it means. You mean it will fail if you don't list it, omit it from here. Or will it fail anyway